.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,21 @@
-**Detailed description**:
+# Description
 
-**Which issue(s) this PR fixes**:
-Fixes #<issue number>
+*Please include a summary of the changes and the related issue. Please also include relevant motivation and context,
+including:*
 
-**Special notes for your reviewer**:
+- What does this PR do?
+- Why are these changes needed?
+- How were these changes implemented and what do they affect?
 
-**Checklist**
-- [ ] Documentation added
-- [ ] Tests updated
\ No newline at end of file
+*Use [Github semantic linking](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to address any open issues this PR relates to or closes.*
+
+Fixes # (issue number, *if applicable*)
```

Comment:
Can this be returned to previous `<issue number, if applicable>`, so it does not show at all